routes/habit.js

```diff
@@ -11,8 +11,13 @@ const router = express.Router();
 const { authenticateUser } = authentication;
 const { authorizeHabitOwnerOrAdmin, authorizeHabitOwner } = authorization;
 const {
+  ensureParamsAreNotEmpty,
   ensureNameIsProvided,
   ensureNameIsNotEmpty,
+  ensureNumberDaysAreProvided,
+  ensureNumberDaysNotEmpty,
+  ensureExpiryDateProvided,
```

Comment:
This answers my earlier question. SO it means that `allowNull` in the models should be false then.